Tests/MapboxMapsTests/Location/LocationManagerTests.swift

```diff
@@ -37,4 +37,27 @@ internal class LocationManagerTests: XCTestCase {
         XCTAssertNil(locationManager.delegate)
         XCTAssertFalse(locationManager.showUserLocation)
     }
+
+    func testLocationManagerPuckUpdate() {
```

Comment:
I'm thinking through the things that this tests…

- if `showUserLocation` is true and the puck type is modified, the location puck manager's puckType is updated
- if `showUserLocation` is changed from true to false, the location puck manager is nil
- if `showUserLocation` is changed from false to true, the location puck manager is non-nil and its puck type is set appropriately
- The default value for showUserLocation is false

Could we separate this test into one smaller test for each of those things?

Reply:
Thanks, updated